docs/index.html

```diff
@@ -42,6 +42,7 @@ <h2 class="title">A 1x Engineer...</h2>
         <li>Appreciates code reviews from their peers that are constructive and helpful.</li>
         <li>Is open to learning.</li>
         <li>Asks for help when they need it.</li>
+        <li>Believes all other engineers are 10x engineers.</li>
```

Comment:
I think framing this less as a self-degrading statement and more as a self-assessing statement would be awesome.

Here's a suggestion, feel free to merge it directly, tweak it, or write your own:

```suggestion
        <li>Can feel like an impostor at times.</li>
```

Reply:
As a 1x engineer I can't do anything else but accept your constructive feedback <3